vehicles/src/modules/special-auth/profile/loa.profile.ts

```diff
@@ -185,157 +203,25 @@ export class LoaProfile extends AutomapperProfile {
 
       createMap(
         mapper,
-        UpdateLoaDto,
         LoaDetail,
+        ReadLoaDto,
         forMember(
-          (d) => d.loaId,
-          mapWithArguments((_, { loaId }: { loaId: number }) => {
-            return loaId;
-          }),
-        ),
-        forMember(
-          (d) => d.documentId,
-          mapWithArguments((_, { documentId }: { documentId: string }) => {
-            if (documentId) return documentId;
-          }),
-        ),
-        forMember(
-          (d) => d.isActive,
-          mapWithArguments((_, { isActive }: { isActive: boolean }) => {
-            return isActive;
-          }),
-        ),
-        forMember(
-          (d) => d.expiryDate,
+          (d) => d.companyId,
           mapFrom((s) => {
-            return s.expiryDate;
-          }),
-        ),
-        forMember(
-          (d) => d.updatedUserGuid,
-          mapWithArguments((_, { userGUID }) => {
-            return userGUID;
-          }),
-        ),
-        forMember(
-          (d) => d.updatedUser,
-          mapWithArguments((_, { userName }) => {
-            return userName;
-          }),
-        ),
-        forMember(
-          (d) => d.updatedUserDirectory,
-          mapWithArguments((_, { directory }) => {
-            return directory;
+            return s.company?.companyId;
           }),
         ),
         forMember(
-          (d) => d.updatedDateTime,
-          mapWithArguments((_, { timestamp }) => {
-            return timestamp;
+          (d) => d.startDate,
+          mapFrom((s) => {
+            return dayjs(s.startDate).format('YYYY-MM-DD');
```

Comment:
Just curious. Is this required. What is the response without mapping? Does it add a timestamp without it?

Reply:
done